app/docker-compose.yml

```
@@ -38,6 +38,8 @@ services:
       - PHOENIX_TLS_KEY_FILE_PASSWORD=
       # For verifying client certificates
       - PHOENIX_TLS_VERIFY_CLIENT=False
+      # In case we want to disable PII Redaction
+      - REDACT_PII=True
```

Use the value set in `local.env`
```suggestion
      - REDACT_PII=${REDACT_PII}
```